build.gradle

```diff
@@ -5,14 +5,41 @@ buildscript {
     }
 }
 
+plugins {
+    id 'com.jfrog.bintray' version '1.2'
+    id 'net.researchgate.release' version '2.1.2'
+}
+
 allprojects {
     apply plugin: 'idea'
+    apply plugin: 'java'
+    apply plugin: 'scala'
+    apply plugin: 'maven'
+    apply plugin: 'maven-publish'
+    apply plugin: 'com.jfrog.bintray'
 }
 
-subprojects {
+release {
```

Comment:
yeah thats correct, I realized that only after failing inside of the subprojects block many times.